server/src/main/java/org/candlepin/controller/ContentManager.java

```diff
@@ -324,12 +324,15 @@ public Content updateContent(ContentData update, Owner owner, boolean regenerate
 
         if (contentData == null || contentData.isEmpty()) {
             // Nothing to import
-            return new HashMap<String, Content>();
+            new ImportResult<Content>(Collections.<String, Content>emptyMap(),
```

Comment:
Creation of empty ImportResult is repeated quite often. What about creating non-parametric constructor that will construct ImportResult with empty maps?